apps/designer/app/designer/features/style-panel/shared/use-style-data.ts

```diff
@@ -58,13 +59,10 @@ export const useStyleData = ({
     () => cssRule?.style as Style
   );
 
-  const currentStyle = useMemo<Style>(
-    () => ({
-      ...selectedInstanceData?.browserStyle,
-      ...breakpointStyle,
-    }),
-    [selectedInstanceData?.browserStyle, breakpointStyle]
-  );
+  const currentStyle = useStyleInfo({
+    localStyle: breakpointStyle,
+    browserStyle: selectedInstanceData?.browserStyle ?? {},
```

Comment:
selectedInstanceData can only be undefined if there is no selected instance, in this case we shouldn't have have to call useStyleInfo at all

Reply:
having a default there makes this call and the logic inside useStyleInfo confusing because we can't have an empty browserStyle actually

Reply:
It's just the check is done later and since this is a hook we can't call it after check. So I'd say good for now. Later we will improve this.

Reply:
yeah I know, but it's still confusing to have a default object there, I think maybe having undefined inside useStyleInfo is more logical?